test/cypress/e2e/send.spec.js

```diff
@@ -162,29 +162,44 @@ describe('Send', () => {
   });
 
   /**
-   * Initialization dialogue and functionality is shown for the fresh account
+   * Initialization dialogue and functionality shouldn't be shown through wallet send.
```

Comment:
The annotation here does not correlate the test title. It should be about the positive case showing the banner, not negative about not showing the dialogue in send